src/Deck.java

```diff
@@ -0,0 +1,165 @@
+import java.util.Random;
+import java.util.List;
+import java.util.ArrayList;
+
+/**
+ * Created by patrickwert on 3/20/17.
+ */
+public class Deck {
+    ArrayList<Tile> Deck = new ArrayList<Tile>();
```

Comment:
You can probably just declare it here `ArrayList<Tile> Deck;` since we initialize it down there?